Topic7/src/main/java/com/aguirre/topic7/models/CourseDao.java

```diff
@@ -0,0 +1,7 @@
+package com.aguirre.topic7.models;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface CourseDao extends JpaRepository<Course, Long> {
+
+}

```

Comment:
Because I like more JPARepository (the method findAll() returns a List, the other returns a Iterable)